api-playground/troubleshooting.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: 'Troubleshooting'
+description: 'Common issues with API References'
+icon: 'triangle-exclamation'
+---
+
+API pages are complicated. As a result, there are a lot of things that can go wrong.
+Here's a list of common issues we've seen customers run into:
+
+<AccordionGroup>
+  <Accordion title="All of my OpenAPI pages are completely blank">
+    In this scenario, it's likely that either Mintlify cannot find your OpenAPI document,
+    or your OpenAPI document is invalid.
+
+    Running `mintlify dev` locally should reveal some of these issues.
+
+    To verify your OpenAPI document will pass validation:
+
+    1. Visit [this validator](https://apitools.dev/swagger-parser/online/)
+    2. Switch to the "Validate text" tab
+    3. Paste in your OpenAPI document
+    4. Click "Validate it!"
+
+    If the text box that appears below has a green border, your document has passed validation.
+    This is the exact validation package Mintlify uses to validate OpenAPI documents, so if your document
+    passes validation here, there's a great chance the problem is elsewhere.
+  </Accordion>
+
+  <Accordion title="One of my OpenAPI pages is completely blank">
+    This is usually caused by a misspelled `openapi` field in the page metadata. Make sure
+    the HTTP method and path match the HTTP method and path in the OpenAPI document exactly.
+    If you are specifying a particular OpenAPI file in the `openapi`, ensure the filepath
+    and file extension are correct.
```

Comment:
would it be good to add an example of what this might look like too?

Reply:
done!